cloudinit/tests/test_util.py

```diff
@@ -730,6 +730,36 @@ def already_mounted_device(self, already_mounted_device_and_mountdict):
         """already_mounted_device_and_mountdict, but return only the device"""
         return already_mounted_device_and_mountdict[0]
 
+    @mock.patch("platform.system")
+    @mock.patch("cloudinit.util.subp.subp")
+    @mock.patch("cloudinit.temp_utils.tempdir")
+    def test_normalize_vfat_on_bsd(self, m_tmpdir, m_subp, m_is_BSD):
```

Comment:
This test could be expanded via parameterisation to also exercise the "msdosfs" codepath (and perhaps also combined with the below test). Let me know if you'd like more details and I can run you through it!